Engine/AssemblyFinder.cs

```diff
@@ -97,6 +97,8 @@ private void SyncFiles()
                             {
                                 if (StrEq(subDir.Name, "obj"))
                                     continue; // skip obj subfolder
+                                if (subDir.GetFiles("Tap.exe").Any())
+                                    continue; // skip subdirectory if it appears to be an opentap installation
```

Comment:
`tap.exe` only exists on Windows. We should probably check `OpenTap.dll` instead.